src/assets/stylesheets/info-dialog.scss

```diff
@@ -3,8 +3,7 @@
   height: 100%;
   top: 0;
   left: 0;
-  position: absolute;
-  pointer-events: none;
+  position: fixed;
```

Comment:
I notice the `<body>` scrolling issue (from [my GIF](https://github.com/mozilla/hubs/pull/368#issuecomment-387236650)) still remains - want me to file a follow-up issue for that?

Reply:
Yep. I mentioned in slack, the rest of these fixes are clearly minimal impact but I could imagine that one having unintended side-effects since it involves putting `pointer-events: none` on the `body`